.github/2.0/cookbooks/Daemon.md

```diff
@@ -525,6 +446,85 @@ assert client.pods.delete(pod_id)
 
 ------
 
+## RESTful Flows
+
+JinaD enables management of (remote + containerized) Flows with all your dependencies via REST APIs.
+
+<details>
+<summary><strong>Create a Flow (<a href="https://api.jina.ai/daemon/#operation/_create_flows_post">redoc</a>)</strong></summary>
+
+This creates a new container using the base image & network defined by `workspace_id` & starts a Flow inside the container. Only the ports needed for external communication are mapped to local. Make sure you've added all your config files while creating the workspace in the previous step.
+
+```python
+from daemon.clients import JinaDClient
+client = JinaDClient(host=HOST, port=PORT)
+client.flows.create(workspace_id=workspace_id, filename='my_awesome_flow.yml')
+# jflow-a71cc28f-a5db-4cc0-bb9e-bb7797172cc9
+```
+
+</details>
+
+<details>
+<summary><strong>Get details of a Flow (<a href="https://api.jina.ai/daemon/#operation/_status_flows__id__get">redoc</a>)</strong></summary>
+
+```python
+client.flows.get(flow_id)
+```
+
+<details>
+<summary>Example response</summary>
+
+```text
```

Comment:
Can we have JSON syntax highlight?

Reply:
This actually returns a python dict, and adding json highlighting in the markdown looks weird for python dict.